hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRevokeKycFromAccountHandler.java

```diff
@@ -90,13 +90,13 @@ public void handle(@NonNull TransactionBody txn, @NonNull WritableTokenRelationS
         tokenRelStore.put(tokenRelBuilder.build());
     }
 
-    private void pureChecks(TokenRevokeKycTransactionBody op) {
+    private void preCheck(TokenRevokeKycTransactionBody op) throws PreCheckException {
```

Comment:
You can retain the name as `pureChecks`. This will be just validating `TransactionBody` without needing dynamicProperties or state. We will have one more method that will be done as first step in `handle` (need to decide its name). 
That will do some other checks we are doing in `validateSemantics` now, which involve dynamicProperties and/or state

Reply:
Done